doc/source/guidelines/private_packaging.rst

```diff
@@ -0,0 +1,78 @@
+.. _private_dependencies:
+
+
+Hosting Private Dependencies
+============================
+There will be cases in which it is necessary to host and pull packages that are
+not ready to be hosted to the public `PyPI`_. For example, if a PyAnsys library
+requires auto-generated gRPC interface files from an as-of-yet private feature
+or service, this package should be hosted on a private PyPI repository.
+
+Ansys has a private repository at `PyAnsys PyPI`_, and access is controlled via
+a secret PAT, specified in the GitHub secret ``PYANSYS_PYPI_PRIVATE_PAT`` which
+is only available to repositories within the `PyAnsys`_.
+
+.. note::
+   This includes forked repositories. This is designed to protect against PRs
```

Comment:
Not quite obvious here what "includes" means - do forked repos have access, or not? I'm assuming the latter, but the wording might be improved.

Reply:
Addressed in https://github.com/pyansys/dev-guide/pull/65/commits/3e9d81042c8f8f4bbbefdcaa66b2e5f3e328afe6